openwisp_radius/settings.py

```diff
@@ -92,21 +92,21 @@ def get_settings_value(option, default):
             '{} must contain '.format(value)
             + '{organization}, {uid} and {token}'  # noqa
         )
-except AssertionError as e:
+except AssertionError as e:  # pragma: no cover
     raise ImproperlyConfigured(
         'OPENWISP_RADIUS_PASSWORD_RESET_URLS is invalid: {}'.format(str(e))
     )
 
-try:  # pragma: no cover
+try:
     SMS_TOKEN_HASH_ALGORITHM = getattr(hashlib, SMS_TOKEN_HASH_ALGORITHM)
-except ImportError as e:
+except ImportError as e:  # pragma: no cover
     raise ImproperlyConfigured(
         'OPENWISP_RADIUS_SMS_TOKEN_HASH_ALGORITHM is invalid: {}'.format(str(e))
     )
 
-try:  # pragma: no cover
+try:
     assert int(SMS_TOKEN_LENGTH) <= 8 and int(SMS_TOKEN_LENGTH) >= 4
-except AssertionError:
+except AssertionError:  # pragma: no cover
```

Comment:
Move `# pragma: no cover` to except block instead of try block, because try block is cover, I think the intentions of whoever added them was to add it for the except block.